pii-reporting-ui/src/app/features/pii-settings/pii-settings.component.css

```
@@ -0,0 +1,1018 @@
+
+/* ===== THRESHOLD SECTION ===== */
+.threshold-card {
+  padding: 20px;
+  border-radius: 10px;
+  border: 1px solid var(--sentinel-border-default);
+  background: var(--sentinel-bg-surface);
+  margin-bottom: 16px;
+}
+
+/* ::ng-deep needed: PrimeNG inputnumber internal structure */
+.threshold-card ::ng-deep .p-inputnumber .p-inputnumber-input {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: ng-deep

(lint/correctness/noUnknownPseudoElement)